embulk-cli/src/main/java/org/embulk/cli/EmbulkMigrate.java

```diff
@@ -236,7 +237,9 @@ public void copy(String sourceResourcePath, String destinationFileName)
         {
             Path destinationPath = this.basePath.resolve(destinationFileName);
             Files.createDirectories(destinationPath.getParent());
-            Files.copy(EmbulkMigrate.class.getClassLoader().getResourceAsStream(sourceResourcePath), destinationPath);
+            Files.copy(EmbulkMigrate.class.getClassLoader().getResourceAsStream(sourceResourcePath), destinationPath,
+                    StandardCopyOption.REPLACE_EXISTING);
```

Comment:
I don't like this style too. 😄 
Airlift format like that.

Did you mean the following?

```java
Files.copy(EmbulkMigrate.class.getClassLoader().getResourceAsStream(sourceResourcePath), destinationPath,
           StandardCopyOption.REPLACE_EXISTING);
```